supplementary/alica_tracing/src/tracing/Trace.cpp

```diff
@@ -49,6 +49,7 @@ std::string Trace::context() const
 
 void Trace::finish()
 {
+    setTag("endTime", std::chrono::duration_cast<std::chrono::milliseconds>(std::chrono::system_clock::now().time_since_epoch()).count());
```

Comment:
alica uses whatever clock is set via `setClock`, so ideally that same clock needs to be used here as well. One way to do that would be to have a `Clock` template parameter & pass the clock as constructor parameters via `setTraceFactory`, `TraceFactory` & `Trace` classes

Reply:
@Dale-Koenig since it's not clear from the ticket, do we want the time here to be unix time or alica's time?

Reply:
Trace only makes sense as Unix time imo.

Reply:
Then I'll leave things as is @veerajsk, since alica's clock can use ROS time